client/README.md

```diff
@@ -46,43 +46,23 @@ Baseline task combines `clean` and `bootstrap` tasks into a single command:
-
-2. Launch VSCode on the root directory of the project (so in the folder structure you'll have `.vscode` directory from step 1 at the root). You can do this via the GUI or if you've installed VSCode on your path you can navigate to the root and run `code .`.
-
-3. Ensuring you have a typescript file open, open the command palette (Ctrl/Cmd + Shift + P) and search for `TypeScript: Select Typescript Version...`
-![](docs/images/lsp_vscode_one.png)
-> If you don't see this option then ensure you have a `.ts` file open.
-> If you don't see this option, it may be because you need to set VSCode to trust the workspace (so it picks up the config from step 1) when asked.
-
-4. Select the workspace version, which should have the path matching the path set in the value of step 1.
-![](docs/images/lsp_vscode_two.png)
-> If you don't see this option then ensure that you've opened the project in VSCode that has the `.vscode` directory from step 1 at the root.
-
-5. That should be it! Please note that you'll not see any diagnostics information after the LSP had loaded until you interact and change the file.
-
-### JetBrains
-
-This section applies to JetBrains IDEs such as IntelliJ and Webstorm, though the location of the settings menus might slightly differ between versions.
-
-**IMPORTANT**: Currently JetBrains does not leverage the TypeScript server for full IntelliSense in their IDEs so our plugin can only offer limited functionality. You should receive full diagnostics support from the plugin, but only limited definition coverage, and no help with autocompletion. Currently we are in discussion with JetBrains to come to a solution about this.
-
-1. Launch the IDE.
-
-2. Open the preferences menu option from the settings.
-![](docs/images/lsp_jetbrains_one.png)
+```json
+"peerDependencies": {
+  "@genesislcap/foundation-entity-management": "~14",
+  "@genesislcap/foundation-layout": "~14",
+  "@genesislcap/foundation-logger": "~14",
+  "@genesislcap/foundation-ui": "~14",
+  "@genesislcap/foundation-zero": "~14"
+}
+```
 
-3. Navigate to the `Typescript` settings in the `Languages & Frameworks` settings, and ensure that the typescript option is set to the `node_modules/typescript` of your _local_ project, as shown in the image. This may be the default already, in which case you don't need to do anything. You'll also want to enable at least the three options which are enabled in the image below.
-![](docs/images/lsp_jetbrains_two.png)
+Yet the PBC metadata repo and `blank-app-seed` are being run against a pre-release version of foundation,
+like `14.127.6-pbc-demo.4`, then the npm install will fail with an `unable to resolve dependency tree` error.
 
-4. That should be it! Please note that you'll not see any diagnostics information after the LSP had loaded until you interact and change the file.
+Defining `overrides` here in the `blank-app-seed` addresses the problem for now, but in principle it should work without.
```

Comment:
This readme update isn't right - it removes documentation for the end user on how to setup intellisense in the project and replaces it with a note, which is aimed at us on why peer deps were added.

Intellisense instructions should stay as is.  If we want to add a note to ourselves, we can put it inside a Handlebars comment, which will be removed from the generated app.

```
{{!

Note here

}}
```

> Yet if the PBC metadata repo and `blank-app-seed` are being run against a pre-release version of foundation.
Blank app seed doesn't use any pre-release versions. PBCs shouldn't either. Unless you mean for testing purposes, but in that case you can test with local versions, they don't have to be published to NPM

fyi @derekdon

Reply:
Updating to main branch client folder + README as per earlier discussion. Sorry about this bad merge.